resources/js/components/MenuBuilder.vue

```diff
@@ -39,29 +39,29 @@
         </div>
 
         <div class="buttons o1-gap-x-2 md:o1-w-1/3 o1-flex o1-justify-end o1-content-center">
-          <button
+          <span
```

Comment:
span is not a good use of a button, it goes against html standards and wcag requirements.